babeld/src/ubus.c

```diff
@@ -87,9 +89,8 @@ static void babeld_add_xroute_buf(struct xroute *xroute, struct blob_buf *b) {
   blobmsg_close_table(b, prefix);
 }
 
-// Sends an exported routes message on ubus socket, splitting apart IPv4 and
-// IPv6 routes.
-static void babeld_ubus_get_xroutes(struct ubus_context *ctx_local,
+// Sends an exported routes message on ubus socket, splitting apart IPv4 and IPv6 routes.
```

Comment:
Nit: let's try and decide on a wrapping style and stick to it. This changed twice in 2 PRs. I'm fine with either style.

Reply:
I prefer the vs autoformat option. ;) The rebase destroyed the formating. Fixed. :)